client/components/plans/plan-actions/index.jsx

```diff
@@ -159,35 +157,6 @@ var PlanActions = React.createClass( {
 		page( '/checkout/' + this.props.site.slug );
 	},
 
-	handleClickStartTrial: function( event ) {
-		event.preventDefault();
-
-		if ( this.props.isSubmitting ) {
-			return;
-		}
-
-		this.recordStartFreeTrialClick();
```

Comment:
We should also remove this `recordStartFreeTrialClick()` method above - it's not used at all.